appcues/src/main/java/com/appcues/trait/ExperienceTrait.kt

```diff
@@ -2,5 +2,13 @@ package com.appcues.trait
 
 interface ExperienceTrait {
 
+    enum class ExperienceTraitLevel {
+        EXPERIENCE, GROUP, STEP
+    }
```

Comment:
maybe un-nest this? or make it `ExperienceTrait.Level` ?  Just thinking over a bit more since its part of the public API now.

Reply:
I will un-nest it